src/date-runtime.js

```diff
@@ -0,0 +1,55 @@
+define([
+	"./common/runtime-key",
+	"./common/validate/parameter-presence",
+	"./common/validate/parameter-type/date",
+	"./common/validate/parameter-type/string",
+	"./core-runtime",
+	"./date/format",
+	"./date/formatter/fn",
+	"./date/parse",
+	"./date/parser/fn",
+	"./date/tokenizer",
+
+	"./number-runtime"
+], function( runtimeKey, validateParameterPresence, validateParameterTypeDate,
+	validateParameterTypeString, Globalize, dateFormat, dateFormatterFn, dateParse, dateParserFn,
+	dateTokenizer ) {
+
+Globalize._dateFormatterFn = dateFormatterFn;
+Globalize._dateParserFn = dateParserFn;
+Globalize._dateFormat = dateFormat;
+Globalize._dateParser = dateParse;
+Globalize._dateTokenizer = dateTokenizer;
+Globalize._validateParameterTypeDate = validateParameterTypeDate;
+
+Globalize.dateFormatter =
+Globalize.prototype.dateFormatter = function( options ) {
+	options = options || { skeleton: "yMd" };
```

Comment:
Is there any way to avoid duplicating this default (also defined in `src/date.js`)? Otherwise fixing #466 will have to change both, and whatever future changes we'll do have the same problem.